vue/src/components/IngredientComponent.vue

```diff
@@ -15,12 +15,31 @@
                 <span v-if="ingredient.amount !== 0" v-html="calculateAmount(ingredient.amount)"></span>
             </td>
             <td @click="done">
-                <span v-if="ingredient.unit !== null && !ingredient.no_amount">{{ ingredient.unit.name }}</span>
+                <template v-if="ingredient.unit !== null && !ingredient.no_amount">
+                    <template v-if="ingredient.unit.plural_name === '' || ingredient.unit.plural_name === null">
+                        <span>{{ ingredient.unit.name }}
+                    </template>
+                    <template v-else>
+                        <span v-if="ingredient.always_use_plural_unit">{{ ingredient.unit.plural_name}}</span>
+                        <span v-else-if="(ingredient.amount * this.ingredient_factor) > 1">{{ ingredient.unit.plural_name }}</span>
```

Comment:
Seconded for english, values less than one are also often pluralized, so the comparison should be `!= 1` for plural usage.

Reply:
it's 1/2 cup honey though...

Reply:
i guess this comes down to two options

1. not implement pluralization as it simply does not work in any consistent matter across different languages (as we did until now, i personally just use things like `Möhre(n)` in my database
2. implement it the way it is right now in this PR and live with the limitations

I am really not sure and would love some input on this, i am kinda leaning towards leaving out pluralization because
1. the added complexity from a technical standpoint
2. the added complexity to the user (and userinterface)
3. the limitation that we cannot implement it correctly without adding even more complexity
4. many users likely not even using the feature as it is a lot of work to enter all that data

that said for some people who want to have well structured databases its the only way to actually enter data correctly. 

really looking for some feedback here

Also here is an interesting article about how wrong you can be with assumptions about languages and grammar even for something as simple as a name https://www.kalzumeus.com/2010/06/17/falsehoods-programmers-believe-about-names/

Reply:
Sorry for the late response.

I have thought about the topic for a while now and there are maybe only these two options you have mentioned. It is a difficult topic and my information is just from knowing 1 and a half languages.

What do you think about adding a setting, either server-wide (toggled by an administrator) or per space/user to use this kind of pluralization (by default disabled).

This will not remove any complexity from the UI maybe it makes it even worse.

If this is not the solution you are looking for feel free to either provide an other solution or to reject the MR. I am fine with both results.

Reply:
i am sorry for not working on this earlier as well, i will be on vacation for the next few weeks and once i have time afterwards i will look into this again, sorry.